prompts/configs.go

```diff
@@ -169,14 +187,24 @@ func configBaseForm(quickstart *Quickstart) (*QuickstartState, error) {
 	return &nextState, nil
 }
 
-func languageSpecificForms(language string, existingConfig *config.Configuration, configFields []config.SDKGenConfigField, isQuickstart bool) ([]huh.Field, []string, error) {
+func languageSpecificForms(language string, existingConfig *config.Configuration, configFields []config.SDKGenConfigField, isQuickstart bool, sdkClassName string) ([]huh.Field, []string, error) {
 	langConfig := config.LanguageConfig{}
 	if existingConfig != nil {
 		if conf, ok := existingConfig.Languages[language]; ok {
 			langConfig = conf
 		}
 	}
 
+	// Default the packageName to the SDKClassName
+	// TODO: This updates the placeholder but is intended to be a default value
+	// TODO: We should explain what this is by example ie:
+	//    * python: pip install my-company-sdk
+	//    * go: go get github.com/my-company/my-company-sdk
+	//    * typescript: npm install my-company-sdk
+	if isQuickstart && (language == "go" || language == "typescript" || language == "python") {
```

Comment:
Why are you checking language here?

Reply:
In case kebab case is invalid for a particular language, better to stay on the safe side.